static/org.openpsa.widgets/ui.js

```diff
@@ -93,19 +93,26 @@ const org_openpsa_layout = {
         });
     },
 
-    initialize_search: function(providers, current) {
+    initialize_search: function(providers, config) {
         if (typeof providers !== 'object' || providers.length === 0) {
             return;
         }
+        
+        let defaults = {
+            field: '#org_openpsa_search_query',
+            current: providers[0].identifier
+        };
+
+        if (typeof config === 'string'){
+            config = {field: defaults.field, current: config};
+        }
+
+        Object.assign(defaults, config);
```

Comment:
that looks wrong. According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign this copies from `config` to `defaults`

Reply:
I did that at the begining, and it didn't work. this works as expected.

Reply:
```javascript
var defaults = {a: 1, b: 2},
    config = {b: 4};

Object.assign(defaults, config);

console.log(config);
```
Output: 

```json
{b: 4}
```
No default has been merged into config